geoprocessing/src/modules/shapefiles/shapefiles.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { createReadStream, rmdirSync } from 'fs';
+const mapshaper = require('mapshaper');
+const unzipper = require('unzipper');
+
+@Injectable()
+export class ShapeFileService {
+  private readonly logger: Logger = new Logger(ShapeFileService.name);
+  constructor() {}
```

Comment:
Consider injecting the logger, otherwise it won't leverage global settings (like muting logs)

Reply:
Changes applied [here](https://github.com/Vizzuality/marxan-cloud/pull/161/commits/1cf47128646b7c495378b79d8e2dc76ba563eb83)